NEWS.md

```diff
@@ -1,5 +1,6 @@
 # cpp11 (development version)
 
+*Added `named()` and `empty()` methods to r_vector classes, returning true if named/empty (@sbearrows, #182, #186) 
```

Comment:
We should try to follow https://style.tidyverse.org/news.html#common-patterns when formatting news. Maybe these would be better as two separate entries?

* New `x.named()` method to check if a vector is named (@sbearrrows, #186)
* New `x.empty()` method to check if a vector is empty (@sbearrows, #182)

What do you think?

Reply:
Sure! Looks good,  and thanks for the links